lib/device.cpp

```diff
@@ -67,7 +69,26 @@ QString Device::type() const
     return d->type;
 }
 
+Device::TIPO Device::Typedevice(){
+
+ if(d->type=="drm_minor"){
+    return MONITOR;
+    }
 
+    if(d->subsystem=="video4linux"){
```

Comment:
perche il in tybe mi dice che è un usb   generico con il susystem rilevo  video4linux